recipe/recipe.yaml

```diff
@@ -34,61 +38,110 @@ requirements:
-      - test -f $PREFIX/include/siscone/spherical/siscone.h
-      - test -f $PREFIX/include/siscone/spherical/protocones.h
-      - test -f $PREFIX/include/siscone/spherical/momentum.h
-      - test -f $PREFIX/include/siscone/spherical/hash.h
-
-      - cd examples/
-      - $CXX main.cpp options.cpp -o main $CXXFLAGS $LDFLAGS -lsiscone
-      - ./main
-      - $CXX area.cpp options.cpp -o area $CXXFLAGS $LDFLAGS -lsiscone
-      - ./area
-      - $CXX sample.cpp -o sample $CXXFLAGS $LDFLAGS -lsiscone
-      - ./sample
-      - $CXX test.cpp -o test $CXXFLAGS $LDFLAGS -lsiscone
-      - ./test
-      - $CXX times.cpp -o times $CXXFLAGS $LDFLAGS -lsiscone
-      - if: build_platform == target_platform
-        then: ./times
-      - $CXX spherical.cpp -o spherical $CXXFLAGS $LDFLAGS -lsiscone -lsiscone_spherical
-      - ./spherical
+    - if: unix
+      then:
+        - test -f $PREFIX/lib/libsiscone${SHLIB_EXT}
+        - test -f $PREFIX/lib/libsiscone_spherical${SHLIB_EXT}
+        - test -f $PREFIX/include/siscone/config.h
+        - test -f $PREFIX/include/siscone/geom_2d.h
+        - test -f $PREFIX/include/siscone/defines.h
+        - test -f $PREFIX/include/siscone/circulator.h
+        - test -f $PREFIX/include/siscone/area.h
+        - test -f $PREFIX/include/siscone/ranlux.h
+        - test -f $PREFIX/include/siscone/quadtree.h
+        - test -f $PREFIX/include/siscone/protocones.h
+        - test -f $PREFIX/include/siscone/momentum.h
+        - test -f $PREFIX/include/siscone/hash.h
+        - test -f $PREFIX/include/siscone/vicinity.h
+        - test -f $PREFIX/include/siscone/split_merge.h
+        - test -f $PREFIX/include/siscone/siscone.h
+        - test -f $PREFIX/include/siscone/siscone_error.h
+        - test -f $PREFIX/include/siscone/reference.h
+        - test -f $PREFIX/include/siscone/spherical/geom_2d.h
+        - test -f $PREFIX/include/siscone/spherical/vicinity.h
+        - test -f $PREFIX/include/siscone/spherical/split_merge.h
+        - test -f $PREFIX/include/siscone/spherical/siscone.h
+        - test -f $PREFIX/include/siscone/spherical/protocones.h
+        - test -f $PREFIX/include/siscone/spherical/momentum.h
+        - test -f $PREFIX/include/siscone/spherical/hash.h
+        - cd examples/
+        - $CXX main.cpp options.cpp -o main $CXXFLAGS $LDFLAGS -lsiscone
+        - ./main
+        - $CXX area.cpp options.cpp -o area $CXXFLAGS $LDFLAGS -lsiscone
+        - ./area
+        - $CXX sample.cpp -o sample $CXXFLAGS $LDFLAGS -lsiscone
+        - ./sample
+        - $CXX test.cpp -o test $CXXFLAGS $LDFLAGS -lsiscone
+        - ./test
+        - $CXX times.cpp -o times $CXXFLAGS $LDFLAGS -lsiscone
+        - if: build_platform == target_platform
+          then: ./times
+        - $CXX spherical.cpp -o spherical $CXXFLAGS $LDFLAGS -lsiscone -lsiscone_spherical
+        - ./spherical
+    - if: win
+      then:
+        - if exist %PREFIX%\Library\lib\siscone.lib (echo exists) else (exit /B)
```

Comment:
As `test -f` is silent on success, should this follow the same and only provide output if the file is missing?

Reply:
I think I can replace this with windows commandline's `call` with no argument to achieve that behavior. You have to stick some executable statement in the second arg to `if exist`, otherwise it errors.

Reply:
Cool - that worked swimmingly in the latest CI. Resolving this thread.